src/themes/elcom.ts

```diff
@@ -21,7 +22,72 @@ const grey = {
   900: "#000000",
 } as const;
 
-export const theme: Theme = createTheme({
+const breakpoints = ["xs", "md", "md"] as Breakpoint[];
```

Comment:
Just a comment, why only three, and why md twice? 😅 I see that down below we have xs, sm, md lg and xl?

Reply:
These were the breakpoints for the font specs. The extra md was an error. I auditted the values for the fontSpecs and have seen that it was always the same size going up, in fact there were only two sizes (mobile and desktop). I tweaked the code in https://github.com/visualize-admin/electricity-prices-switzerland/pull/78/commits/c6cc46a114ea2907eaa46aa6144247aa1d5b60e0. Thanks for noticing.